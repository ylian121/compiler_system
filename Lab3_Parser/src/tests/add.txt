# A simple program which adds two numbers together
# This program should output '150'

func main() {
  int a;
  int b;
  int c;
  a = 100;
  b = 50;
  c = a + b;
  print(c);
}
