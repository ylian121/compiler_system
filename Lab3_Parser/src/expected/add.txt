function header: main
{
declare var: a
declare var: b
declare var: c
assign var: a = 100
assign var: b = 50
temp0 = a + b
assign var: c = temp0
print: c
}
